File: CODES/1) C++/2) INTEGER/23.cpp
#include <iostream>

using namespace std;

int main()
{
	int n, soat, min, sec;
	cout << " Sekund = ";
	cin >> n;
	soat = n / 3600;
	n = n % 3600;
	min = n / 60;
	sec = n % 60;
	cout << " Soat = " << soat << endl;
	cout << " Minut = " << min << endl;
	cout << " Sekund = " << sec;
	return 0;
}

File: CODES/3) JAVA SCRIPT/2) INTEGER/23.js
let n = 3700;
console.log(n, " sekund ");
let soat = parseInt(n / 3600);
n %= 3600;
let minut = parseInt(n / 60);
let sekund = n % 60;
console.log(soat, " soat ", minut, " minut ", sekund, " sekund ")